libraries/Microsoft.Bot.StreamingExtensions/OperationException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    internal class OperationException : Exception
+    {
+        public OperationException(string message, int statusCode, object body)
```

Comment:
Why is body an object? Given this is a custom exception build for a very specific purpose, I would expect this to be strongly typed.